src/main/java/frc/robot/subsystems/superstructure/arm/ArmIOReal.java

```diff
@@ -105,7 +121,7 @@ public void config() {
         rightPivotMotor.getConfigurator().apply(rightTalonFXConfiguration);
 
         BaseStatusSignal.setUpdateFrequencyForAll(
-                100,
+                250,
```

Comment:
If we aren't sysiding is there a need to keep this at 250?

Reply:
Probably not.